taxi/forms.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+from django import forms
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    REGEX = r"^[A-Z]{3}\d{5}$"
```

Comment:
The regex pattern `^[A-Z]{3}\d{5}$` correctly enforces the format 'AAA00000' for the license number, as required by the task description. Ensure this pattern is consistently used across all forms where license number validation is needed.